Neos.ContentRepository.Export/src/Processors/AssetExportProcessor.php

```diff
@@ -25,102 +26,71 @@
  */
 final class AssetExportProcessor implements ProcessorInterface
```

Comment:
doesnt have to change here but why is this asset export part of he standalone `Export` package? :D

I mean it says so itself:

> Note: This processor requires the packages "neos/media" and "neos/neos" to be installed!

Reply:
i think because it was all previously hard wired in the export package ... but now it could be moved to neos.neos

Reply:
theoretically it could be put in a completely separate package, but I guess we agree that that is an overkill :)
Can it be in `Neos.Media`? Otherwise, why not just keep it here? It does not create a dependency to `neos/media` it just says: _if_ you want to use this processor, you'll need `neos/media`

Reply:
okay either way it will be a followup pr:) And it DOES imo and php architecture linting would fail

Reply:
cannot be in neos/media but could be in neos/neos. I still think it _should_ be there, this "if you want to use this you got to install XYZ seems fishy"